### ISSUE TITLE
Accessibility - Table cell missing context

### ISSUE THREAD

**Issue by id2win** [GSA/idmanagement.gov#543]
Issue level A

Why?
Sighted users can make a visual association between table header and data cells. This context gives meaning to the data.

Screen reader users must rely on a programmatic relationship being made between header and data cells. As each data cell is read out, the relevant row or column header is also announced. In this way, the data has context.

Recommended fix
Assign columns header for each data cell. 163 occurances across 3 pages. Details for each page lined below - 

- [x] https://my2.siteimprove.com/Inspector/1163140/A11Y/Page?pageId=38247369012&impmd=4047A2DE150B72849C355FDFA2312F26#/sia-r77/failed/bVJwNldiWE0=;ZTdlYjAxOGE=
- [x] https://my2.siteimprove.com/Inspector/1163140/A11Y/Page?pageId=36096484857&impmd=4047A2DE150B72849C355FDFA2312F26#/sia-r77/failed/ckRKN3BhV1k=;NTIxZjA3MTA=
- [x] https://my2.siteimprove.com/Inspector/1163140/A11Y/Page?pageId=38246391907&impmd=4047A2DE150B72849C355FDFA2312F26#/sia-r77/failed/eE4vcHlrbXA=;MWVlOGNiOTQ=
